Sources/Defaults/SwiftUI.swift

```diff
@@ -40,7 +40,7 @@ extension Defaults {
 
 @available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
 @propertyWrapper
-public struct Default<Value: Codable>: DynamicProperty {
+public struct Default<Value: Defaults.Serializable>: DynamicProperty {
```

Comment:
```suggestion
public struct Default<Value: Serializable>: DynamicProperty {
```

Reply:
`@Default` is not in `extension Defaults` this might be typo?

Reply:
Yeah. Never mind.